exec-service-wasmer/src/wasmer_breakpoints.rs

```diff
@@ -148,17 +152,17 @@ pub(crate) fn set_breakpoint_value(instance: &Instance, value: u64) {
     instance
         .exports
         .get_global(BREAKPOINT_VALUE)
-        .expect(format!("Can't get `{}` from Instance", BREAKPOINT_VALUE).as_str())
+        .unwrap_or_else(|_| panic!("Can't get `{}` from Instance", BREAKPOINT_VALUE))
```

Comment:
We should consider not calling `panic!()` here instead returning 0 / 1 as confirmation to the caller. This confirmation value should reach the VM eventually, via the C API, and the VM would halt the execution.

This is a general rule, because `panic!()` stops the entire process and that should be avoided.

Reply:
will address in #13